persist/fs/commitlog/commit_log.go

```diff
@@ -227,26 +238,32 @@ func (l *commitLog) write() {
 			continue
 		}
 
-		now := l.nowFn()
-		if !now.Before(l.writerExpireAt) {
+		if now = l.nowFn(); !now.Before(l.writerExpireAt) {
```

Comment:
Could this not stay declared inline with `if now := l.nowFn(); !now.Before(l.writerExpireAt) { ..`?  It seems now is not used outside of the if clause.